litellm/router_utils/cooldown_callbacks.py

```diff
@@ -44,7 +44,7 @@ async def router_cooldown_event_callback(
     _api_base = (
         litellm.get_api_base(model=_model_name, optional_params=temp_litellm_params)
         or ""
-    )
+    ).rstrip("/")
```

Comment:
why is this needed here? @owaisaamir

Reply:
This uses a different approach to get api_base compared to other prometheus metrics. It will be better to have a standard approach for all metrics.